### PR TITLE
Issue636: fix tense of verbs

### PR DESCRIPTION
**Pull request by Jeff-Wang93**
Tell me if this fix is too jank. I thought of other ways but that would involve more lines of code.

### CODE REVIEW

**Comment by cglewis** [comment]
there's more than just `remove` that needs to be fixed...

**Comment by Jeff-Wang93** [comment]
Roger.

File: vent/menus/main.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
 from vent.menus.services import ServicesForm
 from vent.menus.tools import ToolForm
 
+from vent.helpers.logs import Logger
```
Comment (cglewis) [inline vent/menus/main.py]: this should included (alphabetically) in the import block of vent imports

Comment (Jeff-Wang93) [inline vent/menus/main.py]: oh woops. that's not supposed to be there

File: vent/menus/main.py [hunk 2]
```diff
@@ -217,6 +217,33 @@ def perform_action(self, action):
...
                                      'cores': cores}}
+        # grammar rules
+        vowels = ['a', 'e', 'i', 'o', 'u']
+
+        # consonant-vowel-consonant ending
+        # Eg: stop -> stopping
+        if s_action[-1] not in vowels and \
+           s_action[-2] in vowels and \
+           s_action[-3] not in vowels:
+               form_args['action_dict'] = \
```
Comment (cglewis) [inline vent/menus/main.py]: most of this dict is repeated, why not create the dict first outside the if statements, and then only change the `present_t` key inside the if statements?  that will also probably help with your spacing issue.

Comment (Jeff-Wang93) [inline vent/menus/main.py]: Ah. That's a much better way of doing things. OK

**Pull request merged**